start_jupyter_cm/linux.py

```diff
@@ -19,18 +22,56 @@
 folders = [path for path in sys.argv[1:] if os.path.isdir(path)]
 any_file_selected = len(folders) < len(sys.argv[1:])
 if any_file_selected:
-    subprocess.Popen(["%s/jupyter-%s"])
+    subprocess.Popen(["{path}/jupyter-{terminal}"])
```

Comment:
Is this missing and "f" before the string, i.e. `f"{path}/jupyter-{terminal}"`, or is this a syntax shortcut that I'm not aware of?

Reply:
nvm, I see this is part of an `f` triple quote string... my bad. Stupid Github auto-hide...